infra/build/functions/build_lib.py

```diff
@@ -419,6 +478,9 @@ def get_build_body(steps,
 
   if use_build_pool:
     options['pool'] = {'name': OSS_FUZZ_BUILDPOOL_NAME}
+  if 'env' not in options:
+    options['env'] = []
+  options['env'].append('DOCKER_CLI_EXPERIMENTAL=enabled')
```

Comment:
Can you add a comment on why this is needed?

Reply:
I don't think it's actually needed. buildx used to be an experimental features.